tests/common/configuration/node_config_model.rs

```diff
@@ -48,9 +48,10 @@ impl NodeConfig {
     pub fn new() -> NodeConfig {
         let rest_port = get_available_port();
         let public_address_port = get_available_port();
+        let storage_suffix = rand::thread_rng().gen_range(1, 9999);
```

Comment:
The more tests we add the more often they will collide and fail, right? ;)

Maybe we need a more serious tool, I've found this: https://crates.io/crates/tempfile, could it help?

Reply:
Good point. fixed